lib/capistrano/tasks/npm.rake

```diff
@@ -2,7 +2,7 @@ desc "Install needed node packages"
 task :npm_install, [] => [] do |t, args|
   on roles(:web, :job, :app, :cron) do
     within release_path do
-      execute :npm, :install
+      execute :yarn, :install
```

Comment:
This is the only file I'm unsure about. Is there a way to test this task locally @eanders ? If I understand correctly, this only runs for deployment (invoked in `deploy.rb`)

Reply:
This only runs for deployments done via capistrano (only one at the moment.)  We should be able to remove this entire deployment mechanism come late July.  I can test, though this looks safe enough.

Reply:
ok great, thanks